docs/arcadia-users-group/overview.md

```diff
@@ -21,8 +21,6 @@ The schedule is subject to change.
 | Date         | Tutorial            | Description   |
 | ------------ | ------------------- | ------------- |
 | Mar 5, 2024  | [GitHub Templates: new repos with set structures](20240305-github-templates/lesson.md) | This lesson provides an overview of how to create a new repository from a GitHub template. Using a Python analysis repository as an example, it walks through the function of each file in the template and the benefits that come from using templates.|
```

Comment:
Can you also move this down so it follows reverse chronological order in the table below?

Reply:
fixed